mlflow/transformers/__init__.py

```diff
@@ -988,13 +990,39 @@ def _try_load_model_with_device(model_instance, model_path, device, conf):
     return model
 
 
+def _try_load_model_instance_from_code_paths(path, flavor_config):
+    """
+    Load model instance from saved code_paths folder
+    Currently it only iterates over files in the code_paths folder and tries to import
+    the module, if the saved file is a python module folder then the model instance
+    should be imported from the module directly.
+    """
+    model_type = flavor_config[_PIPELINE_MODEL_TYPE_KEY]
+    code_path = pathlib.Path(path).joinpath(flavor_config[FLAVOR_CONFIG_CODE])
+    for code_file in code_path.iterdir():
```

Comment:
Does this work if the specified code_path is a nnested directory? Like model is defined at `MODEL_PATH/model/modeling_lsg_distilbert.py` and `code_path=MODEL_PATH`.

Reply:
One possible way to avoid the directory issue is saving code path on behalf of users. `code_path` usage is tricky, so probably less troublesome to do it by ourselves?

```
def save_model(...):
   ....

    try:
        transformer.getattr(model.__class__.__name__)
    expect:
        coed_paths.append(inspect.getfile(model.__class__.__name__))
```

Reply:
What if the class depends on other files? I think it's hard to identify all the files from our side. I wrote in the comment, if the code_path is a folder that contains python modules, then they should save it with __init__.py files so that we could directly load class from the folder, folders traversal might be complicated

Reply:
We might want to hold off on this until the rework of code_paths in Q1 (will require a design) and this workaround will likely be moot after that implementation. 
We can wait on supporting complex external model references in code_paths until we have a better story around directory snapshotting and inclusion of arbitrary code (i.e. from a GitHub Repo).

Reply:
@BenWilson2 That makes sense, but we won't be able to provide a workaround for users now?